includes/discovery/sensors/current/junos.inc.php

```diff
@@ -16,7 +16,7 @@
     $multiplier = 1;
     $divisor    = 1000000;
     foreach ($junos_oids as $index => $entry) {
-        if (is_numeric($entry['jnxDomCurrentTxLaserBiasCurrent'])) {
+        if (is_numeric($entry['jnxDomCurrentTxLaserBiasCurrent']) && $entry['jnxDomCurrentTxLaserBiasCurrent'] && $entry['jnxDomCurrentTxLaserBiasCurrentLowAlarmThreshold'] != 0) {
```

Comment:
Is it not worth setting jnxDomCurrentTxLaserBiasCurrentLowAlarmThreshold to null so we can still discovery the sensor here?